tests/bootstrap_helper.js

```diff
@@ -23,5 +23,15 @@ for (const require of window.bootstrapInfo.requires) {
   // previously existed in each chunk entrypoint.
   if (require === 'Blockly.libraryBlocks') {
     window.libraryBlocks = goog.module.get(require);
+    continue;
+  }
+  const exportName = {
+    'Blockly.Dart': 'dartGenerator',
+    'Blockly.Dart.all': 'dartGenerator',
```

Comment:
IIRC the `.all` modules were created to be the entry points for each of the corresponding blocks / generator chunks, back when I was setting up chunked compilation.  They also serve as a handy shortcut for test code that wants to load all of the submodules: just `require` the `.all` module, instead of the base module and each of the submodules separately.

The double entries here are so that some `bootstrap.js`-using scripts can request just the base generator, while others request the whole caboodle.

Reply:
(I realise that to actually answer your question about future-proofing I should have said: "the double entries are here because some bootstrap.js-using scripts (already) request just the base generator, while others (already) request the whole caboodle."